excercise9/itemmaster/ItemMasterDTO.java

```diff
@@ -0,0 +1,96 @@
+package itemmaster;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+public class ItemMasterDTO implements Serializable,Cloneable,Comparable<ItemMasterDTO>{
+	private int item_id;
+	private String item_name;
+	private float item_price;
+	private int unit;
+	
+	private ItemMasterDTO(){
+		System.out.println("Item master object created!");
+	}
+	
+	@Override
+	public int compareTo(ItemMasterDTO o) {
+		return	this.item_id-o.item_id;
+	}
+	
+	private static ItemMasterDTO dto;
+	synchronized public static ItemMasterDTO getInstance() {
+		if(dto==null) {
```

Comment:
Can you please add comments explaining your approach?

Reply:
added comments to all the parts of program